### ISSUE TITLE
500: Internal Server Error Class "Sabre\DAV\Client" not found 

### ISSUE THREAD

**Issue by da-wilky** [LibreCodeCoop/LSNextcloud#4]
Hey,
I know nothing happened here for a long time, I still wanted to try this plugin, but can't get it to work after hours of trying. Can't resolve the error `500: Internal Server Error Class "Sabre\DAV\Client" not found ` that is appearing as soon as I complete a survey (when it should send the data to nextcloud). I tried it with the ZIP File from Github, but also with the cloned repository build with composer. Both are not working in the `limesurvey:lts` version.

Anybody got an idea, how I can fix that issue? Really dont understand why it can't find that file, even tho it is most definetly there (`vendor/sabre/dav/lib/DAV/Client.php`) and the composer should know it:
```json
            "autoload": {
                "psr-4": {
                    "Sabre\\DAV\\": "lib/DAV/",
                    "Sabre\\DAVACL\\": "lib/DAVACL/",
                    "Sabre\\CalDAV\\": "lib/CalDAV/",
                    "Sabre\\CardDAV\\": "lib/CardDAV/"
                }
            },
```
Still I'm absolutly no expert in php/composer stuff.

**Comment by vitormattos** [comment]
Hi @da-wilky 
Did you got the file `LSNextcloud.zip ` ? This file already have the vendor folder.
A strange point that you reported is that you already have the vendor folder and you follow getting the error.

Have a lot of time that I don't work at this code and I do not know if your error is about a newest version of LimeSurvey. Would you expose what's the version of your LimeSurvey?

I don't know if I will have time to do a best look inside this question because I have a big backlog of issues to work on, but GitHub would help to priorize my backlog butting your issue more to top of my backlog using the [sponsor](https://github.com/sponsors/vitormattos) feature.

**Comment by da-wilky** [comment]
Hey :)
I actually found the issue looking at other plugins.
Changing `require 'vendor/autoload.php';` to `require_once(__DIR__ . "/vendor/autoload.php");` solved the issue and makes it possible for me to use the plugin now :)

I encoutered the issue both using limesurvey version `3.28.76` and `6.3.2`. It got solved at least in version `6.3.2`.
Still thanks for your fast reply and hope it helps someone else :)
